inloop/solutions/validators.py

```diff
@@ -0,0 +1,45 @@
+from django.core.exceptions import ValidationError
+from django.dispatch import receiver
+
+from constance import config
+from constance.signals import config_updated
+
+
+def get_allowed_file_endings():
+    """
+    Return all allowed file endings.
+    """
+    return [ending.strip().lower() for ending in config.ALLOWED_FILE_ENDINGS.split(",")]
+
+
+allowed_file_endings = get_allowed_file_endings()
```

Comment:
Remove this module-level variable and retrieve the list of allowed file names once at the beginning of `validate_filenames()`.